modules/nextflow/src/main/groovy/nextflow/splitter/JsonSplitter.groovy

```diff
@@ -70,10 +70,8 @@ class JsonSplitter extends AbstractTextSplitter {
     @Override
     protected Map<String,?> validOptions() {
         def baseOptions = super.validOptions()
-        return [
-            path: String,
-            limit: baseOptions.limit
-        ]
```

Comment:
I think this has been done on purpose because `splitJson` does only support `path`. What cause the error?

Reply:
Try running `main.nf`:
```groovy
Channel.of('[1,null,["A",{}],true]')
| splitJson()
| view
```
with
```shell
NXF_VER=23.05.0-edge nextflow run .
```

Returns

```
N E X T F L O W  ~  version 23.05.0-edge
Launching `./main.nf` [lethal_kay] DSL2 - revision: 2bdd9780c3
ERROR ~ Unknown argument 'into' for operator 'Json' -- Possible arguments: path, limit

 -- Check script 'main.nf' at line: 2 or see '.nextflow.log' file for more details
```

Perhaps we just need to add `into` and `autoClose`:

```groovy
 return [
    into: baseOptions.into,
    autoClose: baseOptions.autoClose,
    path: String,
    limit: baseOptions.limit
]
```

Reply:
I tried multiple examples added to the docs in the PR that created the channel operator. None of the ones I tried works 😢